specification/booking-and-referral.yaml

```diff
@@ -1,6 +1,63 @@
 openapi: 3.0.0
 info:
-  description: This is a work in progress NHS Booking and Referral API </br> **Do NOT build against this specification**
+  description: |
+    ## Overview
+    Use this API to redirect requests to the appropriate backend based on an identifier header.
+    
+    The API is synchronous.
+    
+    The following describes the end-to-end process:
+    1. The sending system sends the payload to the Booking and Referrals API, along with an identifier header.
+    2. The Booking and Referrals API directs the payload to the target backend associated with the identifier header.
+    3. The sending system recieves the response.
+    
+    ## Who can use this API
+    This API can only be used where there is a legal basis to do so. Make sure you have a valid use case before you go too far with your development. You must do this before you can go live (see ‘Onboarding’ below).
```

Comment:
Has onboarding started for this API? I cannot see the "Onboarding" section in the overview section?  
If not then remove the (see ‘Onboarding’ below).